packages/vsce/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to the "cics-extension-for-zowe" extension will be documente
 
 ## Recent Changes
 
+- Enhancement: Added the zowe.cics.showAllCommandsInPalette setting to control the visibility of CICS commands in the command palette. [#285] (https://github.com/zowe/cics-for-zowe-client/issues/285)
```

Comment:
That setting is only for us to toggle them for testing, we don't want to advertise that to users. Instead we should explain the effect it has on them, which is something like the following
```suggestion
- Removed commands from command palette where a suitable context menu option exists. [#285] (https://github.com/zowe/cics-for-zowe-client/issues/285)
```